rollup.config.js

```diff
@@ -27,5 +31,9 @@ export default {
       include: ['node_modules/**'],
     }),
     serve('dist'),
+    production &&
+      strip({
+        include: '**/*.(ts|js)',
```

Comment:
```suggestion
        include: '**/*.[ts|js]x?',
```

tsx, jsx도 포함하도록 바꿔보면 어떨까요 ?.?

Reply:
아주 좋은 생각입니다.